Missing lib folder on published package

# The problem

I have installed this package from npm with `yarn add ast-i18n`. 

Tried to execute the cli with `yarn ast-i18n --src=packages/web/src` and end up with this:

```
yarn ast-i18n --src=packages/web/src
yarn run v1.19.1
$ /Users/renan/www/foton/confy/node_modules/.bin/ast-i18n --src=packages/web/src
internal/modules/cjs/loader.js:775
    throw err;
    ^

Error: Cannot find module '../lib/cli.js'
Require stack:
- /Users/renan/www/foton/confy/node_modules/ast-i18n/bin/cli
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:772:15)
    at Function.Module._load (internal/modules/cjs/loader.js:677:27)
    at Module.require (internal/modules/cjs/loader.js:830:19)
    at require (internal/modules/cjs/helpers.js:68:18)
    at Object.<anonymous> (/Users/renan/www/foton/confy/node_modules/ast-i18n/bin/cli:3:1)
    at Module._compile (internal/modules/cjs/loader.js:936:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:947:10)
    at Module.load (internal/modules/cjs/loader.js:790:32)
    at Function.Module._load (internal/modules/cjs/loader.js:703:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:999:10) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/Users/renan/www/foton/confy/node_modules/ast-i18n/bin/cli' ]
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

# The why's

The cli is as follows.

```js
#!/usr/bin/env node
require('../lib/cli.js').run();
```

Checking the `node_modules` folder I see this:

![image](https://user-images.githubusercontent.com/43140758/70082207-a3a02980-15e8-11ea-94be-9ec6001e1887.png)

There isn't such `lib` folder. We should generate it along with its type declarations and minified version.

--- a/package.json
+++ b/package.json
@@ -1,12 +1,11 @@
 {
   "name": "ast-i18n",
   "version": "1.0.0",
-  "author": "Sibelius Seraphini <sibeliusseraphini@gmai.com> (https://github.com/sibelius)",
+  "author": "Sibelius Seraphini <sibeliusseraphini@gmail.com> (https://github.com/sibelius)",
   "bin": {
     "ast-i18n": "./bin/cli"
   },
   "dependencies": {
-    "@types/prettier": "^1.16.0",
     "ast-types": "^0.12.2",
     "cosmiconfig": "^5.1.0",
     "diacritics": "^1.3.0",
@@ -44,6 +43,7 @@
     "@types/graceful-fs": "^4.1.2",
     "@types/jest": "^24.0.0",
     "@types/jscodeshift": "^0.6.0",
+    "@types/prettier": "^1.16.0",
     "@types/react": "^16.8.2",
     "@types/shelljs": "^0.8.2",
     "@types/yargs": "^12.0.8",
@@ -61,5 +61,6 @@
     "start": "yarn b src/index.ts",
     "test": "jest",
     "watch": "babel --extensions \".es6,.js,.es,.jsx,.mjs,.ts,.tsx\" -w -d ./lib ./src"
-  }
+  },
+  "files": ["lib", "bin"]
 }

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -2,7 +2,7 @@
   "compilerOptions": {
     /* Basic Options */
     "target": "es2017",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
-    "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
+    "module": "umd",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
      "lib": [                                 /* Specify library files to be included in the compilation. */
        "esnext",
        "dom"

